horizon/templates/service.yaml.j2

```diff
@@ -5,6 +5,7 @@ spec:
   externalIPs: {{ network.external_ips }}
   {% endif %}
   ports:
-    - port: {{ network.port }}
+    - port: 80
```

Comment:
Shouldn't it be the other way around? The {{ network.port }} specifying the exposed port.